CorsixTH/Lua/entities/humanoids/patient.lua

```diff
@@ -39,6 +39,13 @@ function Patient:Patient(...)
   self.pay_amount = 0
   -- To distinguish between actually being dead and having a nil hospital
   self.dead = false
+  -- Indicates that the patient is being put on death. Patient may be busy,
+  -- But as soon as he is deoccupy he will start the dying scenario based on this flag
+  -- Except cases are when he is cured or sent home while being busy
+  self.set_to_die = false
```

Comment:
Again, my brain yearns for `marked_for_death` but I can't think of any reason not to use `set_to_die`